### PR TITLE
Ft/wisdom/founders terms policy

### PR DESCRIPTION
**Pull request by Ajinormotor**
## Description

Please include a summary of the changes and the related issue.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                <Image
+                  src={headline}
+                  alt="background_pics"
+                  className="md:w-[79px] w-[50px] relative h-[5px]"
+                />
+              </div>
+            </div>
+
+            <p
+              className="font-[400] text-[18px] w-full font-epilogue text-[#1B1B1B99]
```
Comment (Blossom1-chike) [inline src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx:36]: confirm the color of this text with that on figma

File: src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              Join us at The Intern Place, and let's build a brighter future
+              together. Whether you're a job seeker ready to take the next step
+              or an employer looking to attract top talent, we're here to help
+              you succeed.
+            </p>
+          </div>
+
+          {/* button */}
+          <div className="">
+            <button
```
Comment (Blossom1-chike) [inline src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx:48]: use button component here

File: src/chunks/founders-template-chunk/founders-all-template-chunk/FoundersAllTemplateChunk.tsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+            </p>
+          </div>
+        </div>
+      </div>
+
+      {/*What we offer */}
+      <div className="flex flex-col items-center lg:my-[5rem] mt-[4rem] md:mt-auto md:my-[3rem]">
+        {/* Heading */}
+        <div className="gap-[10px] py-[3rem]">
+          <h1 className="font-clash font-[600] text-[32px] leading-[38px] text-[#1B1B1B]">
```
Comment (Blossom1-chike) [inline src/chunks/founders-template-chunk/founders-all-template-chunk/FoundersAllTemplateChunk.tsx:140]: move the what we offer and founder section to another page, this file has a lot of lines already

File: src/components/cards/founders-offer-card/FoundersOfferCard.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { FoundersOfferCardProps } from "./FoundersOfferCard.type";
+import Image from "next/image";
+
+const FoundersOfferCard = (props: FoundersOfferCardProps) => {
+  return (
+    <section
+      className="flex flex-col border-[1px] rounded-[20px] border-light-blueb bg-[#FFFFFF]
```
Comment (Blossom1-chike) [inline src/components/cards/founders-offer-card/FoundersOfferCard.tsx:8]: there's a type with this border color, check it out

File: src/chunks/founders-template-chunk/founders-all-template-chunk/FoundersAllTemplateChunk.tsx [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+        <div className="md:w-[350px] md:h-[350px] w-[300px] h-[300px]">
+          <Image src={ceo} alt="ibunkun_pics" className="w-full h-full" />
+        </div>
+
+        {/* text */}
+        <div className="gap-[30px] max-w-[715px] max-h-[324px]  text-center md:text-start  my-2 md:my-0 w-full">
+          <h1 className="font-clash font-[600] text-[24px] leading-[40px] text-light-blue">
+            Founder
+          </h1>
+          <p className="font-clash font-[400] text-[18px] leading-[32px] md:my-[2.2rem] text-grey gap-[10px]">
```
Comment (Blossom1-chike) [inline src/chunks/founders-template-chunk/founders-all-template-chunk/FoundersAllTemplateChunk.tsx:174]: on the design, this font is epilogue, please confirm this

**Review by Blossom1-chike — CHANGES_REQUESTED** [review]
also, link the various pages to their corresponding texts on the footer

File: src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx (hunk 2 above)
Comment (Ajinormotor) [reply src/chunks/founders-template-chunk/founders-hero-section-chunk/FoundersHeroTemplateChunk.tsx:48]: i shoud remove the div container?

**Comment by Ajinormotor** [comment]
made the necessary correction highlights

**Review by Blossom1-chike — APPROVED** [review]
changes reviewed

**Pull request merged**